docs/pages/blog/date-pickers-stable-v5.md

```diff
@@ -0,0 +1,100 @@
+---
+title: 'The Date pickers gets a stable v5 release'
+description: Date pickers are now stable. Here is the summary of what changed.
```

Comment:
How about we give a quick summary in the form of the action we want the reader to take?

```suggestion
description: The beta phase is over. Migrate to the latest version for improved DX, customizability, and API consistency.
```

Reply:
Or just:

```suggestion
description: Migrate to the latest version for improved DX, customizability, and API consistency.
```

Reply:
Yeah, that makes a lot of sense.